### ISSUE TITLE
Are snps located in MHC locus valid?

### ISSUE THREAD

**Issue by Davidseye** [gqi/fastASSET#2]
When I use fastASSET to IBD and another non-immune disease, the result shows that snps founded by fastASSET all in MHC locus.Their Pvalue in IBD are >10^-8, but usually about 10^-4 in another disease. Can I use the result to explore their function , actually I an interested in this region.

**Comment by gqi** [comment]
It should be okay to use fastASSET on MHC. Actually fastASSET does not distinguish one regions vs others. It simply analyzes the SNPs given to it.Sent from my iPhoneOn Dec 2, 2024, at 3:54 AM, Davidseye ***@***.***> wrote:﻿
When I use fastASSET to IBD and another non-immune disease, the result shows that snps founded by fastASSET all in MHC locus.Their Pvalue in IBD are >10^-8, but usually about 10^-4 in another disease. Can I use the result to explore their function , actually I an interested in this region.

**Comment by Davidseye** [comment]
yeah,but with high LD snps in this region, I found most&nbsp; comorbidity study or pleiotrophy study delete the snp inside it. Thanks for your quick reply!



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 2, 2024 23:42 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [gqi/fastASSET] Are snps located in MHC locus valid? (Issue #2)




 It should be okay to use fastASSET on MHC. Actually fastASSET does not distinguish one regions vs others. It simply analyzes the SNPs given to it.Sent from my iPhoneOn Dec 2, 2024, at 3:54 AM, Davidseye ***@***.***&gt; wrote:﻿ 
 When I use fastASSET to IBD and another non-immune disease, the result shows that snps founded by fastASSET all in MHC locus.Their Pvalue in IBD are &gt;10^-8, but usually about 10^-4 in another disease. Can I use the result to explore their function , actually I an interested in this region.

**Comment by gqi** [comment]
That’s a good point. FastASSET identifies pleiotropy in the sense of associations, but does not really tease apart LD from the signal.On Dec 2, 2024, at 8:12 AM, Davidseye ***@***.***> wrote:﻿
yeah,but with high LD snps in this region, I found most&nbsp; comorbidity study or pleiotrophy study delete the snp inside it. Thanks for your quick reply!







---Original---

From: ***@***.***&gt;

Date: Mon, Dec 2, 2024 23:42 PM

To: ***@***.***&gt;;

Cc: ***@***.******@***.***&gt;;

Subject: Re: [gqi/fastASSET] Are snps located in MHC locus valid? (Issue #2)









 It should be okay to use fastASSET on MHC. Actually fastASSET does not distinguish one regions vs others. It simply analyzes the SNPs given to it.Sent from my iPhoneOn Dec 2, 2024, at 3:54 AM, Davidseye ***@***.***&gt; wrote:﻿ 

 When I use fastASSET to IBD and another non-immune disease, the result shows that snps founded by fastASSET all in MHC locus.Their Pvalue in IBD are &gt;10^-8, but usually about 10^-4 in another disease. Can I use the result to explore their function , actually I an interested in this region.
